### DISCUSSION TITLE
Admin panel

### DISCUSSION THREAD

**Discussion by sukhrob-coder** [ShukuraliProgrammer/e-commerce#15] [Q&A]
super user yaratdim, lekin kiraolmayabman
shuni ko'rib berolasizmi

**Comment by ShukuraliProgrammer** [comment]
nega screen tashenchi

**Comment by sukhrob-coder** [comment]
![Screenshot from 2024-05-17 16-39-13](https://github.com/ShukuraliProgrammer/e-commerce/assets/104163417/a7c009c8-e2aa-46d0-8617-325a02eac60c)
mana

**Reply by ShukuraliProgrammer** [reply]
yaratgan userni is_active fieldini python manage.py shell qilib tekshirib kuring

**Reply by sukhrob-coder** [reply]
hozir

**Reply by sukhrob-coder** [reply]
is_active false ekan

**Reply by sukhrob-coder** [reply]
is_active True ni siz qilib quyishiz keri manimcha

**Reply by ShukuraliProgrammer** [reply]
update qilib quydim, main branchdan pull olasiz localizga
